assets/training/model_management/src/run_model_download.py

```diff
@@ -21,9 +24,31 @@ def _get_parser():
     parser.add_argument("--model-id", required=True)
     parser.add_argument("--model-download-metadata", required=True, help="Model source info file path")
     parser.add_argument("--model-output-dir", required=True, help="Model download directory")
+    parser.add_argument("--update-existing-model", required=False, help="Update existing model")
     return parser
 
 
+def validate_if_model_exists(model_id):
+    """Validate if model exists in any of the registries."""
+    registries_list = ["azureml-preview", "azureml", "azureml-meta"]
+
+    for registry in registries_list:
+        try:
+            ml_client_registry = get_mlclient(registry_name=registry)
+        except Exception:
+            continue
+        REG_MODEL_ID = model_id.replace("/", "-")  # model name in registry doesn't contain '/'
+        models = ml_client_registry.models.list(name=REG_MODEL_ID)
+        print(f"models: {models}")
+        if models:
+            raise AzureMLException._with_error(
+                AzureMLError.create(ModelAlreadyExists, model_id=model_id, registry=registry)
+            )
+        else:
+            logger.info(f"Model {model_id} has not been imported into the registry. "
```

Comment:
Could we please show more details to the customer? please refer to the error thrown in the import model notebook.

Reply:
This is from the model import notebook